madmom/audio/filters.py

```diff
@@ -1531,3 +1531,72 @@ def __array_finalize__(self, obj):
         # set default values here
         self.fref = getattr(obj, 'fref', A4)
         self.window = getattr(obj, 'window', self.WINDOW)
+
+
+class SemitoneBandpassFilterbank(object):
+    """
+    Time domain semitone filterbank of elliptic filters as propsed in [1]_.
+
+    Parameters
+    ----------
+    filter_order : int, optional
+        Filter order of elliptic filters.
+    passband_ripple_db : int, optional
+        Passband ripple in [Db]
+    stopband_rejection_db : int, optional
+        Stopband rejection in [Db]
+    q_factor : int, optional
+        Q-factor.
+    sr_from_midi : numpy array, optional
+        Sampling rate for each MIDI note.
+    midi_min : int, optional
+        Lowest MIDI note.
+    midi_max : int, optional
+        Highest MIDI note.
```

Comment:
As above, I prefer to have fmin and fmax given in Hz instead of MIDI note numbers.
I am not sure however how to deal with the `sr_from_midi` parameter. It looks to me as if this should be a function depending on the signal's sample rate anyways.